### PR TITLE
Fix osx build

### PR DESCRIPTION
**Pull request by rohityadavcloud**
Fixes maven build unit tests on osx. (Tested on M1 mac)

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
@blueorangutan package

File: plugins/hypervisors/kvm/src/test/java/org/apache/cloudstack/utils/linux/MemStatTest.java [hunk 1]
```diff
@@ -46,6 +46,7 @@ public void setup() throws Exception {
     @Test
     public void getMemInfoParseTest() {
         MemStat memStat = new MemStat();
+        memStat.refresh();
```
Comment (nvazquez) [inline plugins/hypervisors/kvm/src/test/java/org/apache/cloudstack/utils/linux/MemStatTest.java:49]: I see this is causing issue on linux packaging (also line 60):

````
[ERROR] getMemInfoParseTest(org.apache.cloudstack.utils.linux.MemStatTest)  Time elapsed: 0.005 s  <<< ERROR!
java.lang.IllegalStateException: Scanner closed
	at org.apache.cloudstack.utils.linux.MemStatTest.getMemInfoParseTest(MemStatTest.java:49)
````

Comment (nvazquez) [inline plugins/hypervisors/kvm/src/test/java/org/apache/cloudstack/utils/linux/MemStatTest.java:49]:
```suggestion
        if (!System.getProperty("os.name").equals("Linux")) {
            return;
        }
```

File: plugins/hypervisors/kvm/src/test/java/org/apache/cloudstack/utils/linux/MemStatTest.java [hunk 2]
```diff
@@ -56,6 +57,7 @@ public void getMemInfoParseTest() {
     @Test
     public void reservedMemoryTest() {
         MemStat memStat = new MemStat(1024, 2048);
+        memStat.refresh();
```
Comment (nvazquez) [inline plugins/hypervisors/kvm/src/test/java/org/apache/cloudstack/utils/linux/MemStatTest.java:60]:
```suggestion
        if (!System.getProperty("os.name").equals("Linux")) {
            return;
        }
```

**Review by nvazquez — CHANGES_REQUESTED** [review]

**Comment by rohityadavcloud** [comment]
Thanks for code suggestions @nvazquez it makes sense to skip on OSX.

@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
@blueorangutan test

**Review by nvazquez — APPROVED** [review]
LGTM

**Review by shwstppr — APPROVED** [review]

**Pull request merged**